src/main/java/nl/tudelft/contextproject/saveLoad/LoadScript.java

```diff
@@ -0,0 +1,366 @@
+package main.java.nl.tudelft.contextproject.saveLoad;
+
+import main.java.nl.tudelft.contextproject.camera.Camera;
+import main.java.nl.tudelft.contextproject.camera.CameraSettings;
+import main.java.nl.tudelft.contextproject.presets.Preset;
+import main.java.nl.tudelft.contextproject.script.Script;
+import main.java.nl.tudelft.contextproject.script.Shot;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.EndElement;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+
+/**
+ * Utility class for loading a script from an XML file.
+ * The location this file is to be loaded from is stored in the private
+ * variable {@link #loadLocation}, which can be set using {@link #setLoadLocation}.
+ * 
+ * @since 0.3
+ */
+public final class LoadScript {
+
+    /**
+     * Location of the save file to load from.
+     * This is set to savefile.xml per default.
+     */
+    private static String loadLocation = "savefile.xml";
+
+    /**
+     * XMLEventReader that reads from a save file.
+     */
+    private static XMLEventReader reader;
```

Comment:
@TH3steven Seeing as they are being referenced in Javadoc with `@link`, I thought it would be a good idea to add Javadoc to them.